atd-vze/src/views/Tools/ToolsUploadNonCR3.js

```diff
@@ -736,12 +736,10 @@ const ToolsUploadNonCR3 = () => {
 
       <Modal
         isOpen={modalFeedback}
-        toggle={toggleModalSaveConfirm}
+        toggle={() => setModalFeedback(false)}
```

Comment:
this function was causing the other modal to return when clicking away on the screen because `modalSaveConfirm` was getting set to `true`. now exiting by clicking away actually closes the modal and the other one won't return (yay no infinite close loop)